### PR TITLE
Update info.xml

### PR DESCRIPTION
**Pull request by pmaier1**
Please check if I missed something. The screenshot image is actually wrong as it suggests audio playing capabilities.

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -2,9 +2,21 @@
...
-	<description>Mediaviewer adds image and video viewing capabilities</description>
-	<licence>GPL-2.0</licence>
+	<summary>Explore, show and share your media collections</summary>
+	<description>
+A media viewer that adds image and video viewing capabilities to Files view and Public Links.
+
+**Features**
+
+- Support for a large selection of image and video formats (depending on server setup)
+- Image and video viewer in one
```
Comment (PVince81) [inline appinfo/info.xml]: sounds a bit flat, would remove this line

File: appinfo/info.xml [hunk 2]
```diff
@@ -2,9 +2,21 @@
...
+- Fullscreen, zoomable slideshow view integrated with the Files view and Public Links
+- Image rotation
+- Sort images by name or date
+- Image and video download straight from the slideshow
+- Native SVG support
```
Comment (felixheidecke) [inline appinfo/info.xml]: I would just drop SVG support. It's currently not part of the supported mimeType list:
https://github.com/owncloud/files_mediaviewer/blob/master/src/config.json#L6

Comment (pmaier1) [reply appinfo/info.xml]: Why not? With `gallery` apparently it was.

File: appinfo/info.xml (hunk 1 above)
Comment (pmaier1) [reply appinfo/info.xml]: agree, done.

File: appinfo/info.xml (hunk 2 above)
Comment (PVince81) [reply appinfo/info.xml]: raised https://github.com/owncloud/files_mediaviewer/issues/65 for SVG support

**Comment by PVince81** [comment]
merging this as we'll have SVG support anyway

**Pull request merged**